images/win/scripts/SoftwareReport/SoftwareReport.Android.psm1

```diff
@@ -94,6 +94,32 @@ function Build-AndroidTable {
     }
 }
 
+function Build-AndroidVariablesTable {
```

Comment:
Use the same approach - https://github.com/actions/virtual-environments/pull/2542/files#diff-05c7a119902ce70abf3f19f63064f768f26c20ff31859b4adc5f448613be99d9R190

Reply:
Done, thanks!